src/Main/PasswordVerifier.java

```diff
@@ -0,0 +1,97 @@
+package Main;
+
+public class PasswordVerifier {
+
+
+    public boolean Verify(String password) throws Exception {
+        boolean validpass = false;
```

Comment:
Most development teams adopt a naming convention for variables. Two popular ones are:
Camel Case:
`validPass`
Snake Case:
`valid_pass`

I'm not sure what is most common in Java but I'd recommend choosing one. It makes it easier to differentiate words when reading variables.